python/kfserving/docs/KFServingClient.md

```diff
@@ -16,11 +16,82 @@ The APIs for KFServingClient are as following:
 
 Class | Method |  Description
 ------------ | ------------- | -------------
+KFServingClient | [create_credentials](#create_credentials) | Create Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get the specified KFService|
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
 KFServingClient | [delete](#delete) | Delete the specified KFService |
 
+## create_credentials
+> create_credentials(storage_type, namespace=None, **kwargs)
+
+Create or update `Secret` and `Service Account` for GCS and S3 according to credentials information. Once the `Service Account` created or updated, user can configure the `Service Account` in the [V1alpha1ModelSpec](V1alpha1ModelSpec.md) for kfserving.
+
+The API returns name of the `Service Account`.
+
+### Example
+
+Example for creating GCP credentials.
+```python
+from kfserving import KFServingClient
+
+KFServing = KFServingClient()
+KFServing.create_credentials(storage_type='GCS',
+                             namespace='kubeflow',
+                             credentials_file='/tmp/gcp.json')
+```
```

Comment:
I am assuming namespace is optional. Can we add in comments what are optional fields.